recipes/trogon/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "trogon" %}
+{% set version = "636f114655da84c82f3d31649ee7165605349558" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Textualize/{{ name }}/archive/{{ version }}.zip
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/trogon-{{ version }}.tar.gz
```

Comment:
We recommend the PyPI release URL though b/c that one is usually "cleaner" than GH, which tends to be heavier, with extra unnecessary files not used in the build.

```suggestion
```

Reply:
I'm keeping it commented because I will make the change later to use `pypi` as the url when there's a stable release with the LICENSE file included.

Reply:
If you insist on package a specific commit of a dev release you must follow CFEP-05.